editor/debugger/debug_adapter/debug_adapter_protocol.h

```diff
@@ -99,6 +102,11 @@ class DebugAdapterProtocol : public Object {
 	void reset_stack_info();
 
 	int parse_variant(const Variant &p_var);
+	void parse_object(SceneDebuggerObject &p_obj);
+	const Variant parse_object_variable(const SceneDebuggerObject::SceneDebuggerProperty &p_property);
+
+	std::optional<ObjectID> search_object_id(int p_var_id);
```

Comment:
We [do not allow](https://docs.godotengine.org/en/stable/contributing/development/cpp_usage_guidelines.html#standard-template-library) the use of the standard library, why do you need to differentiate an invalid `ObjectID` from the empty `std::optional` state?

Reply:
I assumed the documentation was referring to containers, strings, and common structures that have been replicated with own implementations.

I didn't notice `ObjectID` had methods to validate its ID, so this can be changed without issue

Reply:
Fixed